docs/modules/general/proto.mdx

```diff
@@ -0,0 +1,335 @@
+---
+id: proto
+title: Protocol Versions
+---
+
+The `proto=""` attribute specifies what iteration of PGM a certain XML document was created for. It also instructs PGM on whether to allow the usage of deprecated or newly introduced features within a map. If the value is lower than the currently recommended proto version, the map will load but the XML may be interpreted in an outdated way.
+
+Mapmakers should always use the latest supported proto version, and this may be required of new maps that are to be added to any map compilation projects, such as [ResourcePile](https://mcresourcepile.github.io).
+
+<div className="table-container">
+  <table>
+    <thead>
+      <tr>
+        <th>Map Element</th>
+        <th>Description</th>
+        <th>Value/Children</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>
+          <label>{`<map> </map>`}</label>
+        </td>
+        <td>The main map node containing the protocol version to be used.</td>
+        <td>
+          <span className="badge badge--secondary">XML Modules</span>
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+##### Map Attributes
+
+<div className="table-container">
+  <table>
+    <thead>
+      <tr>
+        <th>Attribute</th>
+        <th>Description</th>
+        <th>Value</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>
+          <label>proto</label>
+        </td>
+        <td>
+          <span className="badge badge--danger">Required</span>
+          The map XML's protocol version.
+        </td>
+        <td>
+          <span className="badge badge--success">Recommended</span>
+          <label>1.4.2</label>
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+##### Map Protocol Values
+
+<div className="table-container">
+  <table>
+    <thead>
+      <tr>
+        <th>Version</th>
+        <th>Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>
+          <label>1.4.2</label>
+        </td>
+        <td>
+          <span className="badge badge--warning">Forward Compatibility</span>
+          Refer to <a href="#changes-in-142">Changes in 1.4.2</a>.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <label>1.4.1</label>
+        </td>
+        <td>
+          <span className="badge badge--warning">Forward Compatibility</span>
+          Refer to <a href="#changes-in-141">Changes in 1.4.1</a>.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <label>1.4.0</label>
+        </td>
+        <td>
+          Filters, regions, and teams are always referenced by its ID.
```

Comment:
```suggestion
          Filters, regions, and teams are always referenced by its ID (replaces <label>name</label>.
```

Reply:
You forgot the second parenthesis, manually added in [6505f6c](https://github.com/PGMDev/Website/pull/67/commits/6505f6cfcdc0e1eb91600eb37ba0991b054b14a3)